src/app/core/services/app-config/feature-flag.service.ts

```diff
@@ -0,0 +1,15 @@
+import { inject, Injectable } from '@angular/core';
+import { AppConfigService } from '@services/app-config/app-config.service';
+
+@Injectable({ providedIn: 'root' })
+export class FeatureFlagService {
+  private appConfigService = inject(AppConfigService);
+
+  isManualDeletionEnabled(): boolean {
+    const enabled = this.appConfigService.getAppConfig()?.features?.manualDeletion?.enabled;
```

Comment:
Please can you change this to an explicit `=== 'true'` check as environment variables are always read as strings and obviously `"false" == true`

Reply:
done